src/components/Header.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { FormEvent, useEffect, useRef } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.